### DISCUSSION TITLE
Uso de jupyter notebooks

### DISCUSSION THREAD

**Discussion by MiguelVitela** [LaunchX-InnovaccionVirtual/CursoIntroPython#207] [Q&A]
Buenas tardes compañeros, disculpen me surgió una duda algo trivial, para la elaboración de las katas no hay problema si utilizo jupyternotebooks directamente? ' es decir si estar trabajando en VSCode con su respectiva extensión?
De antemano muchas gracias por su atención, buena tarde.

**Comment by alfredo-inzunza** [comment]
Hola. No creo que haya problema, incluso puedes utilizar el editor de código que tu prefieras. Lo importante es que tengas tu repositorio con los notebooks ya resueltos para cada módulo. Si en GitHub tienes tu repositorio con las Katas resueltas, creo que no debería ser inconveniente en qué editor lo hayas hecho.

Saludos
